src/opera/test/data/test_dist_s1_config.yaml

```diff
@@ -28,20 +28,24 @@ RunConfig:
         ProductVersion: "1.0"
         ProgramPath: /bin/mkdir
         ProgramOptions:
-        - -p dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1;
-        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1_DATE-FIRST.tif
+        - -p dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1;
+        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1_DIST-GEN-STATUS-ACQ.tif
           bs=1M count=1;
-        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1_DATE-LATEST.tif
+        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1_DIST-GEN-STATUS.tif
           bs=1M count=1;
-        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1_DIST-STATUS-ACQ.tif
+        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1_GEN-METRIC.tif
           bs=1M count=1;
-        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1_DIST-STATUS.tif
+        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20250102T015857Z_20250224T152115Z_S1_30_v0.1.png
           bs=1M count=1;
-        - dd if=/dev/urandom of=dist_s1_pge_test/output_dir/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1/OPERA_L3_DIST-ALERT-S1_T10SGD_20241103T015902Z_20241204T175000Z_S1_30_v0.0.1_GEN-METRIC.tif
+
+        # TODO: Should I include these in the unit tests? We can't generate them without the confirmation db
```

Comment:
Since we have code that checks for these, I say its definitely prudent to include them in our unit test where we are attempting to simulate the SAS output interface to the best of our understanding